packages/zero-tag-processor/src/utils/cssFnValueToVariable.ts

```diff
@@ -19,9 +19,7 @@ export type PluginCustomOptions = {
   /**
    * Object to pass as parameter to the styled css callback functions.
    */
-  themeArgs?: {
-    theme: Theme;
-  };
+  themeArgs?: Record<string, unknown>;
```

Comment:
```suggestion
  theme?: Record<string, unknown>;
```
? Or is this something else?

Reply:
This was the same `themeArgs` passed from the plugin. But this package will never be consumed directly. So I kept `themeArgs` as is.

Reply:
We only need to make changes in the plugin package for user facing APIs.

Reply:
Ah, this is the adapter towards linaria, right?

Reply:
Yes.

Reply:
Alright yeah, let's keep it like this then.